MVC-DataExploration/Controllers/CustomerController.cs

```diff
@@ -16,5 +16,20 @@ public ActionResult Index()
         {
             return View(db.Customers);
         }
+
+        //POST
+        [HttpPost]
+        public ActionResult Create(Customer customer)
+        {
+            db.Customers.Add(customer);
+            db.SaveChanges();
+            return View("index", db.Customers);
```

Comment:
The Index method does not need to be passed a copy of the Customers dataset. It will get it from the database context directly.